decidim-participatory_processes/app/forms/decidim/participatory_processes/admin/participatory_process_form.rb

```diff
@@ -75,6 +78,10 @@ def participatory_process_group
           Decidim::ParticipatoryProcessGroup.find_by(id: participatory_process_group_id)
         end
 
+        def processes(organization)
+          @processes ||= Decidim::ParticipatoryProcess.where(organization: organization)
```

Comment:
Maybe it could use the `where(organization: current_organization)` and so there's no need to pass the `organization` to the method, as the organization it'll always be the current one, don't you think so?

Reply:
Yay, thanks!